timemachine/fe/single_topology.py

```diff
@@ -250,6 +273,15 @@ def get_num_connected_components(num_atoms: int, bonds: Collection[Tuple[int, in
     return len(list(nx.connected_components(g)))
 
 
+MINIMUM_CHIRAL_ANGLE_FORCE_CONSTANT = 10.0
+
+
+def canonicalize_chiral_atom_idxs(idxs):
+    center, i, j, k = idxs
+    results = [(center, i, j, k), (center, j, k, i), (center, k, i, j)]
+    return sorted(results)[0]
```

Comment:
nit: can retain simpler expression, and possibly replace one more usage in this file https://github.com/proteneer/timemachine/pull/1255/files#diff-0b9bd14a098c7a129a18f14ffb94d84fc8e5c1020d1a55d0e6f53d545c1a4084L383-R456

```suggestion
    return min(results)
```

Reply:
good catch will fix